editor/plugins/org.fusesource.ide.projecttemplates/plugin.xml

```diff
@@ -231,15 +231,25 @@
             	</and>
          	</constraint>
       	</project-facet-version>
-	  
 	  	<action
         	facet="jst.camel"
             id="jst.camel.install"
             type="install">
          	<delegate class="org.fusesource.ide.projecttemplates.util.camel.CamelFacetInstallationDelegate"/>
 			<config-factory class="org.fusesource.ide.projecttemplates.util.camel.CamelFacetDataModelProvider"/>
 		</action>
-		
+		<action
+        	facet="jst.camel"
+        	id="jst.camel.version.change"
+        	type="version-change">
+     		<delegate class="org.fusesource.ide.projecttemplates.util.camel.CamelFacetVersionChangeDelegate"/>
+		</action>		
+		<event-handler

```

Comment:
we should use the not deprecated extension point org.eclipse.wst.common.project.facet.core.listeners